lighthouse-cli/bin.ts

```diff
@@ -75,6 +77,7 @@ const cliFlags = yargs
     'config-path': 'The path to the config JSON.',
     'perf': 'Use a performance-test-only configuration',
     'port': 'The port to use for the debugging protocol. Use 0 for a random port',
+    'timeout': 'The timeout in seconds to wait for the page to load before continuing',
```

Comment:
should we add a warning about large traces and OOM? If you're frustrated with slow page loads (#1464), a large number might be the first thing you try. A full warning may be more appropriate for the readme but more people will see something brief written here.